layer/contracts/definitions.py

```diff
@@ -22,6 +22,8 @@ class FunctionDefinition:
     def __init__(
         self,
         func: Callable[..., Any],
+        args: Sequence[Any],
+        kwargs: Mapping[str, Any],
```

Comment:
why do we need these?

Reply:
These are to enable https://github.com/layerai/roadmap/issues/16